clang/lib/CIR/Dialect/IR/CIRDialect.cpp

```diff
@@ -78,6 +78,10 @@ struct CIROpAsmDialectInterface : public OpAsmDialectInterface {
       os << (boolAttr.getValue() ? "true" : "false");
       return AliasResult::FinalAlias;
     }
+    if (auto bitfield = attr.dyn_cast<mlir::cir::BitfieldInfoAttr>()) {      
+      os << bitfield.getName().str();
```

Comment:
Thanks for adding this. Can we make it a bit more tied to bitfields? We should probably prefix it with `bfi`, so the final thing looks like `#bfi_strname`